core/templates/pages/exploration-player-page/learner-experience/tutor-card.component.html

```diff
@@ -66,7 +67,8 @@
     <div class="conversation-skin-tutor-card-bottom-section" *ngIf="isInteractionInline() && displayedCard.getInputResponsePairs().length > 0">
       <oppia-input-response-pair [data]="displayedCard.getInputResponsePairs()[displayedCard.getInputResponsePairs().length - 1]"
                                  [bottomSection]="true"
-                                 [profilePicture]="profilePicture"
+                                 [profilePicturePngDataUrl]="profilePicturePngDataUrl"
+                                 [profilePictureWebpDataUrl]="profilePictureWebpDataUrl"
```

Comment:
Hmm, should we move the fetching of the image into the input-response-pair? I do not see why we need to load it here and then pass it to the input pair.

Reply:
The reason we are not doing that way because the value of the profile picture depends whether the tutor card is present on the exploration editor page or not which seems more specific thing for the tutor card.

Reply:
Hmm, can you explain it more please?

Reply:
Sure,
In [tutor-card.component.ts](https://github.com/oppia/oppia/blob/cf506ed05312ae3beaf079971b67e0a2061f81fa/core/templates/pages/exploration-player-page/learner-experience/tutor-card.component.ts#L171) file we set the profile picture according to the mode of exploration, whether it is in the edit mode or in preview mode.
There is no problem in shifting the whole profile picture part to the `input-response-pair` we can do that but don't you think this particular part is more relevant to the `exploration-player-page` related and should be here only.
Let me know what do you think we can totally shift this part.

Reply:
Yeah, I think I agree with you, it makes more sense here.